egs/aspire/s5/local/multi_condition/aspire_data_prep.sh

```diff
@@ -22,12 +21,11 @@ tmpdir=`pwd`/data/local/data
 mkdir -p $tmpdir
 if [ $stage -le 0 ]; then
 
-  find $dev_transcript/ -name '*.stm'  > $tmpdir/transcripts.flist
+  find $dev_transcript/ -name 'dev.stm'  > $tmpdir/transcripts.flist
   find $dev_audio/ -name '*.wav'  > $tmpdir/wav.flist
   find $test_audio/ -name '*.wav'  > $tmpdir/wav_test.flist
-  find $eval_audio/ -name '*.wav'  > $tmpdir/wav_eval.flist
 
-  n=`cat $tmpdir/transcripts.flist | wc -l`
+  n=`cut -d' ' -f1 $(cat $tmpdir/transcripts.flist) | uniq | wc -l`
   if [ $n -ne 30 ]; then
     echo "Expected to find 30 transcript files in the aspire_single_dev_transcript directory, found $n"
```

Comment:
Change the error message. We are no longer looking for $n files. Actually just check for dev.stm and throw an error if it is not found at early stages of the script. 

Also it might be better to move all these commands to `aspire_data_sanity_check.sh` script which can also be run individually, and call it from here.

Reply:
Thanks for the comments.
Hmm since we are doing "$(cat $tmpdir/transcripts.flist)" here, we could still count the number of files in the stm. Of course I should check whether the stm exists at an earlier stage.

Reply:
Yep however you decide to write the check ensure that the error message is updated according to the new directory structure, so that the users can work out what went wrong.

Reply:
sure man XD